### PR TITLE
adding glossary terms

### PR DESCRIPTION
**Pull request by cswatt**
Adds a whole bunch of terms to the glossary (beta).

preview link: https://docs-staging.datadoghq.com/cswatt/glossary-entries/glossary/

### CODE REVIEW

File: content/en/glossary/terms/session_(RUM).md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: session
+---
+In Datadog RUM, a session is a type of event. A user session begins when a user starts browsing the web application. It contains high-level information about the user, including browser and device.
```
Comment (apigirl) [inline content/en/glossary/terms/session_(RUM).md:4]:
```suggestion
In Datadog RUM, a session is a type of event. A user session begins when a user starts browsing the web application. It contains high-level information about the user, including their browser and device.
```

File: content/en/glossary/terms/sla.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+title: Service Level Agreement (SLA)
+---
+A SLA is an explicit or implicit agreement between a client and service provider stipulating the client’s reliability expectations and service provider’s consequences for not meeting them.
```
Comment (apigirl) [inline content/en/glossary/terms/sla.md:4]:
```suggestion
An SLA is an explicit or implicit agreement between a client and service provider stipulating the client’s reliability expectations and the service provider’s consequences for not meeting them. 
```

File: content/en/glossary/terms/snmp.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+title: Simple Network Management Protocol (SNMP)
+---
+SNMP is protocol for collecting, organizing, and modifying information about managed devices on IP networks.
```
Comment (apigirl) [inline content/en/glossary/terms/snmp.md:4]:
```suggestion
SNMP is a protocol for collecting, organizing, and modifying information about managed devices on IP networks.
```

File: content/en/glossary/terms/snmp_mib.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+title: SNMP Management Information Base (MIB)
+---
+An SNMP MIB is a collection of definitions for the properties (such as data types, access permissions, etc.) of a managed object.
```
Comment (apigirl) [inline content/en/glossary/terms/snmp_mib.md:4]:
```suggestion
An SNMP MIB is a collection of definitions for a managed object's properties (such as data types, access permissions, etc.)
```

File: content/en/glossary/terms/source.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+---
+title: source
+---
+A log source is where logs are being collected and ingested into Datadog.
```
Comment (apigirl) [inline content/en/glossary/terms/source.md:4]:
```suggestion
A log source is where logs are collected and ingested into Datadog.
```

File: content/en/glossary/terms/space_aggregation.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+title: space aggregation
+---
+Space aggregation splits a single metric into multiple timeseries by tags such as host, container, and region. There are four aggregation options: sum, min, max, and avg.
```
Comment (apigirl) [inline content/en/glossary/terms/space_aggregation.md:4]:
```suggestion
Space aggregation splits a single metric into multiple timeseries by tags such as host, container, and region. There are four aggregation options: `sum`, `min`, `max`, and `avg`.
```

File: content/en/glossary/terms/span.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+---
+title: span
+---
+A logical unit of work in a distributed system for a given time period. Multiple spans construct a trace.
```
Comment (apigirl) [inline content/en/glossary/terms/span.md:4]:
```suggestion
A span is a logical unit of work in a distributed system for a given period. Multiple spans construct a trace.
```

File: content/en/glossary/terms/span_id.md [hunk 8]
```diff
@@ -0,0 +1,4 @@
+---
+title: span ID
+---
+A numerical identifier generated by the tracing library for a span. Together with trace IDs, they are used to correlate traces and logs in Datadog.
```
Comment (apigirl) [inline content/en/glossary/terms/span_id.md:4]:
```suggestion
A span ID is a numerical identifier generated by the tracing library for a span. Together with trace IDs, they are used to correlate traces and logs in Datadog.
```

File: content/en/glossary/terms/span_tag.md [hunk 9]
```diff
@@ -0,0 +1,4 @@
+---
+title: span tag
+---
+tags that are applied to spans, in the form of key-value pairs, to correlate a request with other telemetry, to to filter it in searches
```
Comment (apigirl) [inline content/en/glossary/terms/span_tag.md:4]: this one looks like it got cut off too

File: content/en/glossary/terms/slo.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+---
+title: Service Level Objective (SLO)
+---
+A SLO is a target percentage for application performance over a specific period of time.
```
Comment (apigirl) [inline content/en/glossary/terms/slo.md:4]:
```suggestion
An SLO is a target percentage for application performance over a specific period of time.
```

File: content/en/glossary/terms/ssrf.md [hunk 11]
```diff
@@ -0,0 +1,4 @@
+---
+title: server-side request forgery (SSRF)
+---
+SSRF is type of exploit where an attacker uses a server to access or manipulate information.
```
Comment (apigirl) [inline content/en/glossary/terms/ssrf.md:4]:
```suggestion
SSRF is a type of exploit where an attacker uses a server to access or manipulate information.
```

File: content/en/glossary/terms/trace_id.md [hunk 12]
```diff
@@ -0,0 +1,4 @@
+---
+title: trace ID
+---
+A numerical identifier generated by the tracing library for a trace. Together with span IDs, they are used to correlate traces and logs in Datadog.
```
Comment (apigirl) [inline content/en/glossary/terms/trace_id.md:4]:
```suggestion
The trace ID is a numerical identifier generated by the tracing library for a trace. Together with span IDs, they are used to correlate traces and logs in Datadog.
```

File: content/en/glossary/terms/view_(RUM).md [hunk 13]
```diff
@@ -0,0 +1,4 @@
+---
+title: view
+---
+In Datadog RUM, a view is a type of event. A view event is generated each time a user visits a page of the web application.
```
Comment (apigirl) [inline content/en/glossary/terms/view_(RUM).md:4]:
```suggestion
In Datadog RUM, a view is a type of event. A view event is generated each time a user visits a web application page.
```

**Review by apigirl — COMMENTED** [review]
Only overall comment is related to the conversation in Slack about whether or not something should include the term itself in the definition

File: content/en/glossary/terms/intelligent_retention_filter.md [hunk 14]
```diff
@@ -0,0 +1,4 @@
+---
+title: Intelligent Retention Filter
+---
+A Datadog default retention filter that is always active, keeping a representative proportion of traces, true high latency, and diverse error traces, to help you monitor the health of your applications. It is not random, and so traces only retained by Intelligent Retention are not included in trace metrics
```
[the remark this answers is not in the document]
Comment (cswatt) [reply content/en/glossary/terms/intelligent_retention_filter.md:4]: seems to be lowercase in other docs

**Review by apigirl — APPROVED** [review]

**Pull request merged**